invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.overrides

```diff
@@ -77,3 +77,17 @@
     }
   }
 }
+
+.mobile-table-cell .ui.dropdown{
```

Comment:
this change should be in dropdown.overrides

Reply:
It is on it.